crates/cargo-gpu/src/main.rs

```diff
@@ -63,7 +65,37 @@ mod spirv_cli;
 mod spirv_source;
 mod toml;
 
-fn main() {
+/// Central function to write to the user.
+#[macro_export]
+macro_rules! user_output {
+    ($($args: tt)*) => {
+        #[allow(
+            clippy::allow_attributes,
+            reason = "`std::io::Write` is only sometimes called??"
+        )]
+        #[allow(
+            clippy::useless_attribute,
+            reason = "`std::io::Write` is only sometimes called??"
+        )]
+        #[allow(
+            unused_imports,
+            reason = "`std::io::Write` is only sometimes called??"
+        )]
```

Comment:
Does the `allow` annotation allow you to write multiple lints with one reason? Like: 

```rust
#[allow(clippy::allow_attributes, unused_imports, clippy::useless_attribute, reason = "...")]
```

Reply:
Ohhh yes, it does.